config.example.toml

```diff
@@ -38,5 +38,3 @@ password = '...'  # second password
 # This is optional. If present, a button to submit your converted Signal sticker pack
 # to signalstickers.com will be displayed.
 signalstickers_api_token = '...'
-# whether to really propose sticker packs for realsies or to create PR branches but no PRs
-signalstickers_api_test_mode = true
```

Comment:
what happened to test mode?

Reply:
The backend has no test mode. I hope to soon release the backend source, so it can be tested locally. I added a config for the API baseurl for this purpose.

Reply:
So how would I go about testing the API? Would I need to run a staging instance of the backend server?

Reply:
Either run your own local version, or test with the live API. I've merge 50+ packs from Adhesive yesterday, so you can test the "duplicate" with them.